modules/st2-pack-icon/style.less

```diff
@@ -0,0 +1,44 @@
+@size: 22px;
```

Comment:
Hey, wait a second, who said it is 22? Nope. Despite @gmedrano artistic vision, 32px icons are non negotiable.